tests/unit/relays/test_contact_display.py

```
@@ -290,14 +290,18 @@ def test_compute_contact_display_data_no_intelligence_data(self):
             )
         
         # Should handle gracefully when no intelligence data exists
-        # Version-related fields are always present even with no intelligence data
+        # Version-related fields and family_support_counts are always present
         intelligence = result['operator_intelligence']
-        self.assertEqual(len(intelligence), 3)
+        self.assertEqual(len(intelligence), 4)
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)